keyboards/cutie_club/keebcats/gina/info.json

```diff
@@ -0,0 +1,71 @@
+{
+    "keyboard_name": "Keebcats Gina",
+    "manufacturer": "Cutie Club",
+    "url": "",
+    "maintainer": "Cutie Club",
```

Comment:
```suggestion
    "maintainer": "Cutie Club",
    "processor": "atmega32u4",
    "bootloader": "atmel-dfu",
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true
    },
```

Reply:
my apologies, thank you for the review! 

I forgot to migrate the configuration to info.json, this has now been completed :smiley: